cluster/centos/.gitignore

```diff
@@ -0,0 +1,13 @@
+binaries
+
+master/bin/etcd
+master/bin/etcdctl
+master/bin/kube*
+
+minion/bin/brctl
```

Comment:
I suppose this binary is strictly bundled with CentOS release version, can we install it in every machine instead of providing it here?